zeppelin-server/src/main/java/org/apache/zeppelin/service/NoAuthenticationService.java

```diff
@@ -30,7 +30,7 @@
 import org.slf4j.LoggerFactory;
 
 public class NoAuthenticationService implements AuthenticationService {
-  private static Logger logger = LoggerFactory.getLogger(NoAuthenticationService.class);
+  private static final Logger logger = LoggerFactory.getLogger(NoAuthenticationService.class);
```

Comment:
Rename it to be `LOGGER`, this is our convention for static final variables.

Reply:
In the example in the [Constant names](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) section of the Google java style document, it says that `Logger` is not a constant. Could you check it out again?
> // Not constants
...
static final Logger logger = Logger.getLogger(MyClass.getName());

Reply:
I think the logger in google java style document refer to `java.util.logging.Logger` which is a class and do has method to change its state (e.g. Logger#config). But what zeppelin use is `org.slf4j.Logger` is an interface which has no method to change its state.

Reply:
I changed it.